### DISCUSSION TITLE
scoop --version returns what do I do?

### DISCUSSION THREAD

**Discussion by adgower** [ScoopInstaller/Scoop#5301] [Help and Questions]
Do I need to update extras & main? How? What?

```
PS C:\WINDOWS\system32> scoop --version
Current Scoop version:
v0.3.1 - Released at 2022-11-15

'extras' bucket:
cf62d5a07 (HEAD -> master, origin/master, origin/HEAD) wire: Update to version 3.30.4368

'main' bucket:
59fd0f1bf (HEAD -> master, origin/master, origin/HEAD) jdtls: Update to version 1.19.0-202212191235
```

**Comment by r15ch13** [comment]
Everything is up to date.
For updating see: https://github.com/ScoopInstaller/Scoop/wiki/Quick-Start#updating-scoop

**Reply by adgower** [reply]
thanks for the link

I forgot to mention I ran `scoop update` before and it still says it after. I assume this means its all updated?

**Reply by r15ch13** [reply]
`--version` just reports the current local version. It does not check if there are updates available.
If `scoop update` doesn't report anything then everything is updated.
